Solutions/win32-lib/win32-lib.vcxproj

```diff
@@ -319,7 +319,7 @@
       </PrecompiledHeader>
       <WarningLevel>Level4</WarningLevel>
       <Optimization>Disabled</Optimization>
-      <PreprocessorDefinitions>ZLIB_WINAPI;WIN32;_CRT_SECURE_NO_WARNINGS;WIN32;NOMINMAX;_DEBUG;_WINDOWS;_USRDLL;WINVER=_WIN32_WINNT_WIN7;NOMINMAX;%(PreprocessorDefinitions)</PreprocessorDefinitions>

```

Comment:
Is removing `NOMINMAX` intended here, and few other places ?

Reply:
No, I have no idea why VS did that.

Reply:
Ah, because some are duplicated.